### ISSUE TITLE
Packaging

### ISSUE THREAD

**Issue by drvinceknight** [drvinceknight/pyw#3]
Discuss how to package code properly:

- Why?
- `src/<name>`
- setup.py
- (Possibly pypi)

**Comment by daffidwilde** [comment]
I would also add licensing to this list if PyPI is discussed, and it may be worth discussing the "etiquette" of maintaining something that's open to the public.

**Comment by drvinceknight** [comment]
Good idea, we cover it briefly in the rsd workshop but certainly something we can go over again. https://vknight.org/rsd/extras/licences/
